### PR TITLE
eq-324 additional paradata

### PR DESCRIPTION
**Pull request by hamishtaplin**
### Changes
- added `analytics.js` module to interface with google analytics event tracking
- added hooks in template with data to be sent to analytics
- fires events according to errors generated
- fires events when user clicks guidance toggle
### How to test
- `npm run compile`
- visit a page and submit with errors
- verify events are being caught by analytics
- click on guidance toggle
- verify events are being caught by analytics
### Who can test

@bennpearson

### CODE REVIEW

**Comment by hamishtaplin** [comment]
Be good for @bennpearson to CR and test this once @warrenbailey has done his bit.

**Comment by warrenbailey** [comment]
@bennpearson ready for you to review.

We probably also want one of the backend devs to review my URL changes, so either @LJBabbage or @weapdiv-david

File: app/data/1_0213.json [hunk 1]
```diff
@@ -0,0 +1,512 @@
+{
```
Comment (warrenbailey) [inline app/data/1_0213.json:1]: This file is auto-generated now, you don't need to add it back in. Please remove.

**Comment by LJBabbage** [comment]
Your bits look fine Warren bar the default startup page. After running run_app.sh, you get a url that looks like http://localhost:5000/questionnaire/1/**sid**/introduction

**Pull request merged**